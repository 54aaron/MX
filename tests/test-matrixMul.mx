int main(){
    Matrix d;
    Matrix n;
    Matrix a;
    n = [[1,2],[3,4]];
    a = [[5,6],[7,8]];
    d = a *. n;
    prints("==============================================");
    prints("Matrix a = ");
    print_matrix(a);
    prints("==============================================");
    prints("Matrix n = ");
    print_matrix(n);
    prints("==============================================");
    prints("a *. n = ");
    print_matrix(d);
}
