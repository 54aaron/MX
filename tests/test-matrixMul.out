==============================================
Matrix a = 

[ 5, 6 ]
[ 7, 8 ]
==============================================
Matrix n = 

[ 1, 2 ]
[ 3, 4 ]
==============================================
a *. n = 

[ 23, 34 ]
[ 31, 46 ]
